blob_tx_generator/blob.js

```diff
@@ -109,6 +105,38 @@ async function run(data) {
     console.log(`sending to ${account.address}`)
     const res = await axios.post("http://localhost:8545", req)
     console.log(res.data)
+    if (res.data.error) {
+        process.exit(1)
+    }
+
+    let blob_kzg = null
+    try {
+        let start = (await axios.get("http://localhost:3500/eth/v1/beacon/headers")).data.data[0].header.message.slot - 1
```

Comment:
noticed that `getBlockNumber` is out of sync with the beacon slots? not sure why

Reply:
Estimates, but when testing locally, `getBlockNumber` was returning block 280 when the beacon chain was returning 390. This likely wouldn't happen in CI though as we don't wait that long